R/qza_to_tse.R

```diff
@@ -0,0 +1,107 @@
+#' Generates a TreeSummarizedExperiment (TSE) object from .qza artifacts
+#'\code{\link[TreeSummarizedExperiment:SummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'
+#' Construct a TreeSummarizedExperiment object from multiple 
+#' qiime2 artifacts (.qza). 
+#' '\code{\link[TreeSummarizedExperiment:SummarizedExperiment-class]{TreeSummarizedExperiment}}
+#' Embedded metadata for provenance is not maintained in this function and 
+#' instead \code{read_qza()} should be used.
```

Comment:
Does this still hold after this was added to the TreeSE metadata slot?

Can we say that provenance metadata is available from both \code{read_qza()} and from the metadata slot of the  TreeSummarizedExperiment object `(\code{metadata(tse)})`

Reply:
I adjusted this line, tell m what you think